### PR TITLE
Adds new help emotes

### PR DESCRIPTION
**Pull request by Victor239**
## About The Pull Request
Adds handshaking, headpatting and leg poking. Hugs can still be done if targeting the chest on help intent.

## Why It's Good For The Game
More RP choices.

## Changelog
:cl:
add: Adds handshaking, headpatting and leg poking.
/:cl:

### CODE REVIEW

**Comment by Maldaris** [comment]
>headpats
LEWD

**Comment by Superyodeler** [comment]
Heresy

**Comment by CRITAWAKETS** [comment]
>leg poking

oh god its pre-ERP content

**Comment by TheNeoGamer42** [comment]
why doesn't it do what bee said and if its targeting the legs or groin it says NO ERP

**Comment by Maldaris** [comment]
today is the day of the shitpost PR

What have I done

**Comment by dee-dubya** [comment]
A FIRM HANDSHAKE

**Comment by dinnerwoerror** [comment]
LOOKING THE MANAGER IN THE EYE

File: code/modules/mob/living/carbon/carbon_defense.dm [hunk 1]
```diff
@@ -264,7 +264,7 @@
 			return
 		M.visible_message("<span class='notice'>[M] shakes [src] trying to get [p_them()] up!</span>", \
 						"<span class='notice'>You shake [src] trying to get [p_them()] up!</span>")
-	else
+	else if(M.zone_selected == BODY_ZONE_CHEST)
```
Comment (Superyodeler) [inline code/modules/mob/living/carbon/carbon_defense.dm:267]: No heresy allowed
```suggestion
	else if(M.zone_selected == BODY_ZONE_CHEST || M.zone_selected == BODY_ZONE_PRECISE_GROIN)			// Wasp Edit - Adds more help emotes
```

File: code/modules/mob/living/carbon/carbon_defense.dm [hunk 2]
```diff
@@ -295,6 +295,21 @@
 				SEND_SIGNAL(src, COMSIG_ADD_MOOD_EVENT, "friendly_hug", /datum/mood_event/betterhug, M)
 		for(var/datum/brain_trauma/trauma in M.get_traumas())
 			trauma.on_hug(M, src)
+	else if(M.zone_selected == BODY_ZONE_HEAD)
```
Comment (Superyodeler) [inline code/modules/mob/living/carbon/carbon_defense.dm:298]:
```suggestion
	else if(M.zone_selected == BODY_ZONE_HEAD || M.zone_selected == BODY_ZONE_PRECISE_EYES || M.zone_selected == BODY_ZONE_PRECISE_MOUTH)		// Wasp Edit Begin - Adds more help emotes
```

File: code/modules/mob/living/carbon/carbon_defense.dm [hunk 3]
```diff
@@ -295,6 +295,21 @@
...
+			to_chat(M, "<span class='warning'>[src] does not have a [M.zone_selected == BODY_ZONE_L_ARM ? "left" : "right"] arm!</span>")
+		else
+			M.visible_message("<span class='notice'>[M] shakes [src]'s hand.</span>", \
+						"<span class='notice'>You shake [src]'s hand.</span>")
+	else if((M.zone_selected == BODY_ZONE_L_LEG) || (M.zone_selected == BODY_ZONE_R_LEG))
+		if(!get_bodypart(check_zone(M.zone_selected)))
+			to_chat(M, "<span class='warning'>[src] does not have a [M.zone_selected == BODY_ZONE_L_LEG ? "left" : "right"] leg!</span>")
+		else
+			M.visible_message("<span class='notice'>[M] pokes [src]'s leg.</span>", \
+						"<span class='notice'>You poke [src]'s leg.</span>")
```
Comment (Superyodeler) [inline code/modules/mob/living/carbon/carbon_defense.dm:312]:
```suggestion
						"<span class='notice'>You poke [src]'s leg.</span>")		// Wasp Edit End
```

**Pull request merged**